### PR TITLE
Match and Page Style Changes

### PR DESCRIPTION
**Pull request by awilli70**
## Match.html - Basically finished
image

Bottom of page:
image

## Index.html - Finished

image

Image Slideshow with link to dogs:
image

### CODE REVIEW

**Comment by awilli70** [comment]
There's a lot in this PR, so I'll mark out some notable bits of code

File: docs/index.html [hunk 1]
```diff
@@ -7,13 +7,94 @@
     <title>TinDog</title>
     <link rel="stylesheet" href="./styles/styles.css">
     <script src="https://code.jquery.com/jquery-3.6.0.js" integrity="sha256-H+K7U5CnXl1h5ywQfKtSj8PCmoN9aaq30gDh27Xc0jk=" crossorigin="anonymous"></script>
-    <style>#sidenav {display: none;}</style>
+    <style>#sidenav {display: none;}
```
Comment (awilli70) [inline docs/index.html:10]: This line is super important - On page load, it makes sure that the dropdown menu is closed

File: docs/index.html [hunk 2]
```diff
@@ -7,13 +7,94 @@
...
+                left: 0px;
+                opacity: 0.60;
+            }
+            h1 {
+                position: relative;
+                color: #fff;
+                font-size: xx-large;
+                text-align: center;
+                
+            }
```
Comment (awilli70) [inline docs/index.html]: This block (and the media tag following it) are the title bar - for coherent formatting this should be in every file as well

**Comment by awilli70** [comment]
The easiest way to start a page is to use index.html as a starter, removing the slideshow styles and irrelevant jquery
That ensures that 

a) the dropdown doesn't break
b) the background image header is consistent

**Review by bmcder17 — APPROVED** [review]

**Pull request merged**